src/modules/layout/containers/LocationHeader.js

```diff
@@ -30,7 +33,9 @@ type PropsType = {|
   t: TFunction,
   cityName: string,
   isEventsEnabled: boolean,
+  isNewsEnabled: boolean,
```

Comment:
I think it would be more intuitive if we would pass `isLocalNewsEnabled` instead of the combined `isNewsEnabled`. Otherwise one does not really now imo if its just local news or both.

Reply:
The `isNewsEnabled` represent both `localNews` and `tunews`. I mean it's for showing and hiding  the navigation `news` link. calling it `isLocalNewsEnabled` wouldn't be accurate I think.

Reply:
I didn't mean to just change the name of it, but rather just pass `isLocalNewsEnabled` and `isTunewsEnabled` separately instead of doing `isNewsEnabled = isLocalNewsEnabled || isTuNewsEnabled` in the `LocationHeaderContainer`. Makes the code here way clearer imo.